draft-ietf-ppm-dap.md

```diff
@@ -3030,20 +3208,36 @@ Each batch mode specifies the following:
 ## Time Interval {#time-interval-batch-mode}
 
 The time-interval batch mode is designed to support applications in which
-reports are collected into batches grouped by an interval of time. The Collector
-specifies a "batch interval" that determines the time range for reports included
-in the batch. For each report in the batch, the time at which that report was
-generated (see {{upload-flow}}) MUST fall within the batch interval specified by
-the Collector.
-
-Typically the Collector issues queries for which the batch intervals are
-continuous, monotonically increasing, and have the same duration. For example,
-the sequence of batch intervals `(1659544000, 1000)`, `(1659545000, 1000)`,
-`(1659546000, 1000)`, `(1659547000, 1000)` satisfies these conditions. (The
-first element of the pair denotes the start of the batch interval and the second
-denotes the duration.) However, this is not a requirement--the Collector may
-decide to issue queries out-of-order. In addition, the Collector may need to
-vary the duration to adjust to changing report upload rates.
+reports are grouped by an interval of time. The Collector specifies a "batch
+interval" into which report timestamps must fall.
+
+The Collector can issue queries whose batch intervals are continuous,
+monotonically increasing, and have the same duration. For example:
+
+~~~ tls-presentation
+[
+  struct {
+    start = 1659544000,
+    duration = 1000,
+  } Interval,
+  struct {
+    start = 1659545000,
+    duration = 1000,
+  } Interval,
+  struct {
+    start = 1659546000,
+    duration = 1000,
+  } Interval,
+  struct {
+    start = 1659547000,
+    duration = 1000,
+  } Interval
+]
+~~~
```

Comment:
This is longer than the existing example of batch intervals, but uses a syntax we already introduced. I can go either way with it.